index.html

```diff
@@ -211,5 +211,56 @@ <h1 class="about-heading">About me</h1>
         </li>
       </ul>
     </section>
+    <section class="contact-section">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form
+        class="form-control"
+        action="https://formspree.io/f/mzboleyl"
+        method="post"
+      >
+        <input
+          placeholder="Full Name"
+          class="text-field"
+          type="text"
+          maxlength="30"
+          required
+        />
+        <input
+          placeholder="Email Address"
+          class="text-field"
+          type="email"
+          required
+        />
+        <textarea
+          class="message-field"
+          placeholder="Enter text here"
+          maxlength="500"
+        ></textarea>
+        <button type="submit" class="btn-primary">Get In Touch</button>
+      </form>
```

Comment:
- [OPTIONAL] when I tried to submit the form I got this screen, so kindly make sure to add the `name` attribute in all your form elements to work as expected. 💯

![image](https://user-images.githubusercontent.com/45231276/177502536-aca727f5-06cb-43bc-8063-a1f63f9321dc.png)